docs/source/user-guide/configs.md

```diff
@@ -19,46 +19,74 @@
 
 # Configuration
 
-## BallistaContext Configuration Settings
+## Ballista Configuration Settings
 
-Ballista has a number of configuration settings that can be specified when creating a BallistaContext.
+Configuring Ballista is like configuring DataFusion. Apart from a few Ballista specific configurations all others are the same like DataFusion.
```

Comment:
nit: I think this can be rephrased to be more concise:

```suggestion
Configuring Ballista is quite similar to configuring DataFusion. Most settings are identical, with only a few configurations specific to Ballista.
```

Reply:
added